tasks/install_powershell.json

```diff
@@ -42,6 +42,14 @@
       "description": "The number of retries in case of network connectivity failures",
       "type": "Optional[Integer]",
       "default": 5
+    },
+    "username": {
+      "description": "The username to use when downloading from a source location requiring authentication",
+      "type": "Optional[String]"
+    },
+    "password": {
+      "description": "The password to use when downloading from a source location requiring authentication",
+      "type": "Optional[String]"
```

Comment:
https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/769#discussion_r2025214649

Reply:
Updated for both powershell and bash